### PR TITLE
DCO can not be retrieved using Public API.

### PR DESCRIPTION
**Pull request by sampadsaha5**
Story: https://www.notion.so/ADSINT-16-5e74ffb062fc44eba2bedfdefb20cec2

### CODE REVIEW

**Comment by sampadsaha5** [comment]
Investigation : https://smartnews.slack.com/archives/CP273PW4X/p1657005180518539?thread_ts=1656305900.772089&cid=CP273PW4X

File: smartnews-ads-management-api.md [hunk 1]
```diff
@@ -169,6 +170,7 @@ Retrieve some or all campaigns associated with the current account.
 #### GET /v1.0/campaigns/{campaignId}
 
 Retrieve details for a specific campaign.
+Note: Campaigns that has Dynamic Creative can not be retrieved through this API.
```
Comment (rkverma3395) [inline smartnews-ads-management-api.md:173]:
```suggestion
Note: Campaigns that has Dynamic Creative ON can not be retrieved through this API.
```

Comment (sampadsaha5) [inline smartnews-ads-management-api.md]: 👍

**Comment by sampadsaha5** [comment]
Got LGTM from PdM.
https://smartnews.slack.com/archives/CP273PW4X/p1664785163121849?thread_ts=1656305900.772089&cid=CP273PW4X

**Review by rkverma3395 — APPROVED** [review]
LGTM

**Review by chihchenghuang — APPROVED** [review]
LGTM

**Review by patrick-atienza — COMMENTED** [review]
LGTM

**Review by patrick-atienza — COMMENTED** [review]
LGTM

**Pull request merged**